### PR TITLE
Restore player properties on cancel

### LINKED ISSUE #52: Canceling editing a player's info keeps their edited values.
- create a player without a height or weight
- click the edit icon
- enter a height and weight
- click cancel
- weight and height show the height and weight

MIght be related to #49 

cursor_and_localhost_5000_dev_html__players

We might need the `backup` plugin from CanJS.

### PR DESCRIPTION
**Pull request by akagomez**
- Call `this.backup()` on `Player.init`
- Call `player.restore()` on `player-edit .cancel-btn click`
- Test

This PR depends on commits in PR #72. 

Closes #52.

### CODE REVIEW

File: public/components/player/edit/edit_test.js [hunk 1]
```diff
@@ -109,5 +115,76 @@ QUnit.test("Cancel event", function(assert){
...
+	var editedWeight = 210;
+	var editedHeight = 67;
+
+	var player = new Player({
+		name: initialName,
+		weight: initialWeight,
+		height: initialHeight
+	});
+
+	var frag = can.stache('<player-edit player="{player}"></player-edit>')({
```
Comment (akagomez) [inline public/components/player/edit/edit_test.js]: Change binding syntax.

File: public/components/player/edit/edit_test.js [hunk 2]
```diff
@@ -109,5 +115,76 @@ QUnit.test("Cancel event", function(assert){
...
+				weight: editedWeight,
+				height: editedHeight
+			});
+		});
+
+	F('#player-name').val(editedName, 'Edited name is correct');
+	F('#player-weight').val(String(editedWeight), 'Edited weight is correct');
+	F('#player-height').val(String(editedHeight), 'Edited height is correct')
+
+	F('player-edit .cancel-btn').click('Editing is cancelled')
```
Comment (akagomez) [inline public/components/player/edit/edit_test.js]: Could had been a pure VM test. Call cancel directly.

**Comment by justinbmeyer** [comment]
@akagomez somehow this did not land. Can you check your branches and see what happened?

**Comment by akagomez** [comment]
@justinbmeyer I'm not sure how this happened. My best guess is that a rebase clobbered it. The commit log for this PR is puzzling. 

That said, I did eventually find the original commit: https://github.com/donejs/bitballs/commit/6a864f13c0041b132f66b4b2652ea182634e6d3f

I'll open a new PR with those changes reapplied.

**Pull request merged**